src/wazuh_modules/inventory_sync/src/inventorySyncFacade.hpp

```diff
@@ -256,21 +262,23 @@ class InventorySyncFacadeImpl final
                             throw InventorySyncException("Invalid message type");
                         }
                     }
-                    // m_indexerConnector->registerNotify(
-                    //     [this, ctx = res.context]()
-                    //     {
-                    //         // Send ACK to agent.
-                    //         m_responseDispatcher->sendEndAck(Wazuh::SyncSchema::Status_Ok, ctx);
-                    //         // Delete data from database.
-                    //         m_dataStore->deleteByPrefix(std::to_string(ctx->sessionId));
-                    //         // Delete Session.
-                    //         if (m_agentSessions.erase(ctx->sessionId) == 0)
-                    //         {
-                    //             std::cerr
-                    //                 << "InventorySyncFacade::start: Session not found, sessionId: " << ctx->sessionId
-                    //                 << std::endl;
-                    //         }
-                    //     });
+
+                    // Register notify to be called when the indexer is flushed.
+                    m_indexerConnector->registerNotify(
+                        [this, ctx = res.context]()
+                        {
+                            // Send ACK to agent.
+                            m_responseDispatcher->sendEndAck(Wazuh::SyncSchema::Status_Ok, ctx);
+                            // Delete data from database.
+                            m_dataStore->deleteByPrefix(std::to_string(ctx->sessionId));
+                            // Delete Session.
+                            if (m_agentSessions.erase(ctx->sessionId) == 0)
+                            {
+                                std::cerr
```

Comment:
Do we want this console output directives here? Shouldn't we use loggerHelper class

Reply:
We implemented the logging in #31313